akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -78,6 +78,12 @@ for cases when the actor constructor takes value classes as arguments.
 
 @@@
 
+@@@ note { title=DIY }
```

Comment:
btw, looks like the fiddle example is placed in the middle of something else. Move it up above `### Props` and start with 

Here is another example that you can edit and run in the browser: